R/zzz.R

```diff
@@ -0,0 +1,17 @@
+# Stuff to set up on load, run this last (hence the filename)
+# Taken from googlesheets3/zzz.R
+.onLoad <- function(libname, pkgname) {
+
+  op <- options()
+  op.meetupr <- list(
+    meetupr.httr_oauth_cache = TRUE,
+    meetupr.client_id = "blah",     # replace this
```

Comment:
@GregSutcliffe Should this be called `client_key` since they use the term "key"?  

I am also new to OAuth terminology but it seems like they use the word "consumer", so maybe these are the "consumer key" and "consumer secret"?  Do you think it would be more appropriate to use `consumer_key` and `consumer_secret`?

Reply:
That's probably fair, and clearer - let me go change that...

Reply:
Committed that change, thanks for the feedback!